client/components/map/index.jsx

```diff
@@ -44,12 +45,13 @@ const map = ({
         {...getCurrentStyle(active, feature)}
         onClick={() => actions.map.setActive(feature._id)}
       >
-        <Tooltip 
-        distance={actions.distance} 
-        trailName={(feature.properties.NAME || feature.properties.name)}
-        isActive={getActive(feature, active)}
-        {...getActive(feature, active)}
-          />
+        <Tooltip
+          id={feature._id}
+          distance={actions.distance}
+          distances={distances}
```

Comment:
Fixed to not pass all distances. Originally we were passing all, and the id for the popup to lookup its own. Now we look up the distance for it, and if it does not exist, it will generate its own.